src/widgets/DataTable/DataTableRow.js

```diff
@@ -228,6 +229,7 @@ const DataTableRow = React.memo(
                 chevron_right: ChevronRightIcon,
                 history: () => <HistoryIcon color={SUSSOL_ORANGE} />,
                 pencil: () => <PencilIcon color={SUSSOL_ORANGE} />,
+                breach: () => <HazardIcon color={SUSSOL_ORANGE} />,
```

Comment:
the default color is `SUSSOL_ORANGE` and so doesn't need to be specified. It does however look neater having all three lines look the same :)

Reply:
damnit mark - I couldn't decide what to do. So I removed it! 95fbce7d251693867724fec0e8534afc71f3e41c